plugins/node/opentelemetry-instrumentation-pg/src/utils.ts

```diff
@@ -153,10 +153,19 @@ export function handleConfigQuery(
     Array.isArray(queryConfig.values)
   ) {
     try {
-      span.setAttribute(
-        AttributeNames.PG_VALUES,
-        JSON.stringify(queryConfig.values)
-      );
+      const convertedValues = queryConfig.values.map(value => {
+        if (!value) {
+          return 'null';
+        } else if (value instanceof Buffer) {
+          return value.toString();
+        } else if (typeof value === 'object') {
+          return JSON.stringify(value);
```

Comment:
Should we try to use a potential existing `toPostgres` function like PG does?

In general a try/catch around this conversions is likely a good idea as `JSON.stringify` could throw.

Reply:
Ok. added 'toPostgres' condition.

We already have a try/catch for all this conversion, but I added a printout of the error itself now.